engine/src/flutter/impeller/entity/contents/line_contents.cc

```diff
@@ -188,4 +168,47 @@ std::optional<Rect> LineContents::GetCoverage(const Entity& entity) const {
   return geometry_->GetCoverage(entity.GetTransform());
 }
 
+std::vector<uint8_t> LineContents::CreateCurveData(Scalar width,
```

Comment:
Since the curve data is always a given size, you might use a std::array instead of a growable vec

Reply:
Using the std::vector allows us to keep the actual size inside of the .cc file so we can tweak it and only have to recompile 1 .cc file (plus the unit test file currently).  So it's a tossup which is better.